src/screens/ConfirmationContainer/index.js

```diff
@@ -13,15 +13,22 @@ type Props = {
   /**
    * @desc Function to return the Promise resolve
    */
-  onSuccess: (number) => null,
+  onSuccess: (number, number) => null,
```

Comment:
Please add names to parameters and also description in docs, like `onSuccess: (name1: number, anotherName: number)`

Reply:
changed